tests/scanners/test.langpack.js

```diff
@@ -4,61 +4,52 @@ import PropertiesParser from 'parsers/properties';
 import DoctypeParser from 'parsers/doctype';
 import FluentParser from 'parsers/fluent';
 
-import { unexpectedSuccess } from '../helpers';
-
 
 describe('LangpackScanner', () => {
   it('should report a proper scanner name', () => {
     expect(LangpackScanner.scannerName).toEqual('langpack');
   });
 
-  it('scan property files', () => {
-    const spy = sinon.spy(PropertiesParser.prototype, 'parse');
+  it('should scan property files', async () => {
     const code = 'foo = bar';
-    const scanner = new LangpackScanner(code, 'foo.properties');
+    const langpackScanner = new LangpackScanner(code, 'foo.properties');
+
+    sinon.spy(PropertiesParser.prototype, 'parse');
```

Comment:
@hameleonka do you mind this sinon.spy back to the start of this test?
I guess that LangpackScanner was not supposed to call it in its constructor (or the test would have failed an assertion), but I think that it is safer to move it before the creation of the LangpackScanner instance (and also: if we wanted to ensure that the PropertiesParser.prototype.parse method is never called in the constructor, this way we could check that the spy has not be called yet right after the line that creates the instance).